include/rpm/rpmfileutil.h

```diff
@@ -30,6 +30,14 @@ typedef enum rpmCompressedMagic_e {
     COMPRESSED_ZSTD		= 10	/*!< zstd can handle */
 } rpmCompressedMagic;
 
+/** \ingroup rpmfileutil
+ * RPM glob flags, largely modelled after glob(3).
+ */
+typedef enum rpmglobFlags_e {
+    RPMGLOB_NONE		= 0,
+    RPMGLOB_NOCHECK		= (1 << 0), /*!< like GLOB_NOCHECK */
```

Comment:
I find "nocheck" a very strange name for what it does. How about just RPMGLOB_FALLBACK or something?

Reply:
:smile: My words exactly. I kept the name of the similar flag used in `glob(3)` and I actually wanted to just re-use it, but having our own type here means we can indeed come up with a better name. Noted.